Individual package READMEs (#8012)
mistercrunch/Airflow: airflow/providers/google/BACKPORT_README.md

<!--
 Licensed to the Apache Software Foundation (ASF) under one
 or more contributor license agreements.  See the NOTICE file
 distributed with this work for additional information
 regarding copyright ownership.  The ASF licenses this file
 to you under the Apache License, Version 2.0 (the
 "License"); you may not use this file except in compliance
 with the License.  You may obtain a copy of the License at

   http://www.apache.org/licenses/LICENSE-2.0

 Unless required by applicable law or agreed to in writing,
 software distributed under the License is distributed on an
 "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->

##  Backport Google Package limitations:

> * Kubernetes Pod operators imports are refactored to contrib.kubernetes

Reviewer: I don't understand that. Can you tell more about it?

Author: Fixed. Please check if it 's better now :)

> * BigQuery operator is removed until it is refactored.

Reviewer 2: ```suggestion
* BigQuery operators are removed until it is refactored.
```

Reviewer 2: Does this mean that when I install backport packages I will have no BQ operators?

Reviewer: This is the expected result. I think it is worth adding information that we recommend using airflow.contrib operators and that new ones will be released soon.

Reviewer 2: I agree about adding this info. Otherwise, as a user I would suspect that after installation I will have no BQ operators, including those from contrib.

Author: Updated. Please check :)

<!--
 Licensed to the Apache Software Foundation (ASF) under one
 or more contributor license agreements.  See the NOTICE file
 distributed with this work for additional information
 regarding copyright ownership.  The ASF licenses this file
 to you under the Apache License, Version 2.0 (the
 "License"); you may not use this file except in compliance
 with the License.  You may obtain a copy of the License at

   http://www.apache.org/licenses/LICENSE-2.0

 Unless required by applicable law or agreed to in writing,
 software distributed under the License is distributed on an
 "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->

> ##  Backport Google Package limitations:

Reviewer: Can you add tables that compare Airflow 1.10 and Airflow 2?  It's best if it was a few tables - one table that describes new operators, the other with moved operators.

Author: CAn you add it please? I think you have an automated way of doing it and I might be mistaken again.

Reviewer: If you want, I can prepare these tables for you.

Author: Please!

> * Kubernetes Pod operators are imported directly from airflow.contrib.

Reviewer: ```suggestion
```
This is not true. We use GKEPodOperator. We never use KubernetesPodOperator.

Author: Please. Describe it as you think is best.

* BigQuery operators are removed until it is refactored. This is going to happen
  soon and we recommend to use the 1.10 BigQuery operators from the airflow.contrib
  package.

> * GKEStartPodOperator is removed as it uses functionality only available in 2.0.

Reviewer: What do you think about something similar to the following text?

> The following operators have not been released:
> * All operators for BigQuery service. 
> * GKEStartPodOperator for Kubernetes Engine service.
> 
> We recommend staying with the old versions yet. These operators will be released as soon as possible.

Author: Feel free to add it.

<!--
 Licensed to the Apache Software Foundation (ASF) under one
 or more contributor license agreements.  See the NOTICE file
 distributed with this work for additional information
 regarding copyright ownership.  The ASF licenses this file
 to you under the Apache License, Version 2.0 (the
 "License"); you may not use this file except in compliance
 with the License.  You may obtain a copy of the License at

   http://www.apache.org/licenses/LICENSE-2.0

 Unless required by applicable law or agreed to in writing,
 software distributed under the License is distributed on an
 "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->

## Changelog

> ### v2020.XX.XX

Reviewer: @potiuk Can you look at it?

Author: Beautiful! Thank you!

> ### v2020.XX.XX

Reviewer: ```suggestion
### v20.XX.XX
```

Reviewer: The PIP project applies a similar convention.

Author: Yes. I prefer 2020 as well.

This is the first released version of the package.

**Limitations**

The following operators have not been released:

 * All operators for BigQuery service.
 * GKEStartPodOperator for Kubernetes Engine service.

We recommend staying with the old versions yet. These operators will be released as soon as possible.

> **New integration**:

Reviewer: ```suggestion
**New operators**:
```
